omnigibson/systems/macro_particle_system.py

```diff
@@ -332,13 +333,16 @@ class MacroVisualParticleSystem(MacroParticleSystem, VisualParticleSystem):
     """
     Particle system class that procedurally generates individual particles that are not subject to physics
     """
-    # Maps particle name to dict of {obj, link}
+    # Maps particle name to dict of {obj, link, face_id} -- NOTE: face_id will only exist for particles on cloths
     _particles_info = None
 
     # Pre-cached information about visual particles so that we have efficient runtime computations
     # Maps particle name to local pose matrix for computing global poses for the particle
     _particles_local_mat = None
 
+    # Maps group name to array of face_ids where particles are located if the group object is a cloth type
+    _cloth_face_ids = None
```

Comment:
Can you add the expected data structure of `_cloth_face_ids` to the comment above?

Reply:
done